src/main/java/edu/ucsb/mapache/services/TeamEmailListService.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.mapache.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.ucsb.mapache.entities.Student;
+import edu.ucsb.mapache.repositories.StudentRepository;
+
+@Service
+public class TeamEmailListService {
+  @Autowired
+  private StudentRepository studentRepository;
+
+  public String formatTeamEmails(List<Student> students) {
+    String emailList = "";
+    for (Student s : students) {
+      emailList += s.getEmail() + "\n";
+    }
+    return emailList;
+  }
+  public String getTeamEmails(String teamName) {
```

Comment:
Minor, but I was a little confused by the naming of this method when I first saw it. `getTeamEmails` sounds like it should return a list of emails, but this method is actually concatenating emails and returning a human-friendly string when the team is not found (as opposed to returning `null` or raising an exception. Maybe something like `formatTeamEmailsFromTeamName` is better?

Reply:
We have a function called `formatTeamEmails` to format the list of emails. The function of `getTeamEmails` is to get the list of students from the `studentRepository` and call `formatTeamEmails` to get a formated string of email list. It returns a string of email list so that is the reason why it is called `getTeamEmails`.

Reply:
My main issue with the naming is that the method signature, `String getTeamEmails(String)`, doesn't make it clear what the inputs and outputs of the method are.

Again, I realize this is a super nitpicky complaint and we don't have really have an established convention for this in the codebase, so you're free to dismiss this comment. I don't mean to block this PR from getting merged, I just wanted to note that having more self-documenting method signatures would make it easier to read through the code.

Reply:
@btk5h yeah, I agree with you. When I named that function, I didn't know what to name it. As for the name, since we have a different function to format the string, maybe we can change the name to `getEmailsStringFromTeamname`?

Reply:
Sounds good, I think `getEmailsStringFromTeamName` is a solid option.

Reply:
Fixed